BUILDING.txt

```diff
@@ -557,6 +586,22 @@ the zlib 1.2.7 source tree.
 
 http://www.zlib.net/
 
+
+Build command:
+The following command builds all the modules in the Hadoop project and generates the tar.gz file in
+hadoop-dist/target upon successful build. Run these commands from an
+"x64 Native Tools Command Prompt for VS 2019" which can be found under "Visual Studio 2019" in the
+Windows start menu.
+
+> set classpath=
```

Comment:
In markdown I think you can use the triple quote to make it look like a command.

Reply:
Right, but this file (`BUILDING.txt`) isn't rendered as markdown. It should've been `BUILDING.md`.

I'm using the `>` at the beginning to signify the start of the command in a Windows terminal. Like -

![image](https://user-images.githubusercontent.com/10280768/197380892-35ee5d16-ef1d-4336-8146-fcb75f977f45.png)

Reply:
We have this inconsistent plain text, txt, and md files all over the project.
Let's merge this one but we probably want to open a JIRA just to switch everything to markdown.

Reply:
I've created this JIRA - https://issues.apache.org/jira/browse/HADOOP-18509.